src/routes/api.php

```diff
@@ -4,7 +4,7 @@
 
 Route::namespace('LaravelEnso\ControlPanelApi\App\Http\Controllers')
     ->middleware(['auth:api'])
-    ->prefix('token')->as('token.')
+    ->prefix('api/token')->as('token.')
```

Comment:
I don't think that's ok

Reply:
![image](https://user-images.githubusercontent.com/10864136/71964126-3f3d3f00-3212-11ea-9940-31df89a51883.png)
these are the addresses that we expect to exists

Reply:
where that is a mistake too
![image](https://user-images.githubusercontent.com/16073274/71964622-a3f29c80-3206-11ea-83f4-e6f0ce74b59f.png)

Reply:
Oh! :(